### PR TITLE
Added framework for admin_config and admin_status, still needs fleshi…

### PR DESCRIPTION
**Pull request by mitra42**
Framework for admin_config and admin_status as in #36  

But note - still needs fleshing out with some data

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def receive_signal(signal_number, frame):
         if server not in servers:
             servers[server] = '197001010000'
             
-allowable_methods = ['scan_status:POST', 'send_status:POST', 'sync:GET']
+allowable_methods = ['scan_status:POST', 'send_status:POST', 'sync:GET', 'admin_config:GET', 'admin_status:GET']
```
Comment (danaronson) [inline server.py:70]: why not keep the protocol in the doc "admin/config" and "admin/status"?

File: contacts.py [hunk 2]
```diff
@@ -245,6 +245,23 @@ def sync(self, data, args):
...
+            'testing': self.testing
+        }
+        return ret
+
+    # admin_status get
+    def admin_status(self, data, args):
+        ret = {
+            # TODO-36 DAN - how to get some meaningful stat from rtree
+            #'geo_points': self.rtree.count(),
+            'id_len': len(self.ids)
```
Comment (danaronson) [inline contacts.py:261]: we should then probably make len work correctly (this one will only give you a count of the number of unique first two bytes of ids)

File: server.py (hunk 1 above)
Comment (mitra42) [reply server.py:70]: I don't really mind - but all the others were xxx_yyy, *AND* wasn't sure how to catch xxx/yyy in this framework - I don't have a strong opinion either way.

File: contacts.py (hunk 2 above)
Comment (mitra42) [reply contacts.py:261]: Oops - do you want to add methods on Contacts for both of these ?

File: server.py (hunk 1 above)
Comment (danaronson) [reply server.py:70]: i was planning on broaching renaming again to "status/scan" and "status/send" too.

File: contacts.py (hunk 2 above)
Comment (danaronson) [reply contacts.py:261]: yes, will add to this branch before i merge

File: contacts.py [hunk 3]
```diff
@@ -52,28 +53,54 @@ def flush(self):
 # [DATE] is the date it gets entered in the system and [PSEUDORANDOM] is used to differential contacts with the same ID that come in at the same time
 # (accuracy is to minutes).  The date strings are 'YYYYMMDDHHmm'
 
+registry = {}
+
+def register_method(_func = None, *, route):
+    def decorator(func):
+        registry[route] = func
+        def wrapper(*args, **kwargs):
+            print('hi')
```
Comment (mitra42) [inline contacts.py:62]: What is the "print hi" - debugging I presume ?

**Comment by danaronson** [comment]
it was code left over for debugging.  Exactly the kind of thing that
ideally a reviewer should hopefully catch if the developer didn't :-)

**Pull request merged**